include/DgmOctree.h

```diff
@@ -21,11 +21,11 @@
 //#define TEST_CELLS_FOR_SPHERICAL_NN
 
 #ifndef CC_DEBUG
-#ifdef CC_CORE_LIB_USES_QT_CONCURRENT
```

Comment:
Shouldn't your restore that as well? (I don't want to compile the official Windows version with TBB 😉  )

Reply:
Sorry, I forgot to convert the PR to draft before starting to work on this. Should be correct now